share/revng/test/configuration/revng/pypeline-comparison.yml

```diff
@@ -10,7 +10,13 @@ commands:
         filter: example-executable-1 and with-debug-info
     suffix: /
     command: |-
-      "${SOURCES_ROOT}/share/revng/test/tests/pypeline-comparison/prepare.py" "$INPUT" "$OUTPUT";
+      pushd "$OUTPUT" &> /dev/null;
+      cp "$INPUT" input;
+      revng2 project init input;
+      mv model.yml model_init.yml;
+      revng analyze revng-initial-auto-analysis --model=model_init.yml input > model.yml;
+      rm model_init.yml;
+      popd &> /dev/null;
```

Comment:
Comment

Reply:
This got cut down drastically, please check if comments are still needed